### ISSUE TITLE
`multipass find` output not clear enough

### ISSUE THREAD

**Issue by popey** [canonical/multipass#8]
I appreciate we have both `breezy` and `bionic` as codenames, so technically there have been two `b` releases, multipass doesn't seem to know `breezy`. So why complain about multiple 'b' options?

```
alan@hal:~$ multipass launch b
failed to launch: Too many images matching "b"                                  
```
```
alan@hal:~$ multipass find
multipass launch …   Starts an instance of   Image version
----------------------------------------------------------
14.04                Ubuntu 14.04 LTS        20171208
   (or: t, trusty)
16.04                Ubuntu 16.04 LTS        20171208
   (or: default, lts, x, xenial)
17.04                Ubuntu 17.04            20171208
   (or: z, zesty)
17.10                Ubuntu 17.10            20171213
   (or: a, artful)
daily:18.04          Ubuntu 18.04 LTS        20171213
   (or: b, bionic, devel)
```

**Comment by Saviq** [comment]
Right, you need to explicitly say `daily:b` to get at the daily images. We thought it would be clear enough, apparently not...

**Comment by townsend2010** [comment]
I agree that the use of the "remote" is cryptic and we don't specify anywhere what the supported remote names are.  At least 'lxc launch --help' has an example that shows the use of the remote, but I think we still need a way to make the remote names easily discoverable by the user and explain a little better how they are used.

**Comment by townsend2010** [comment]
This has been addressed w/ #170.
